front/src/app/components/common/toggler/index.tsx

```diff
@@ -48,7 +50,7 @@ export class Toggler extends React.PureComponent<ITogglerProps, never> {
             <input
                 key="a"
                 name={name}
-                type="checkbox"
+                type={type || 'checkbox'}
                 className={cssClasses.input}
                 checked={value}
```

Comment:
here i use 'value' as external interface. please refer to comment about using `checked` and `value` together.